mediapipe/tasks/ios/genai/inference/BUILD

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 The MediaPipe Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@build_bazel_rules_swift//swift:swift.bzl", "swift_library")
+
+package(default_visibility = ["//mediapipe/tasks/ios:__subpackages__"])
+
+licenses(["notice"])
+
+swift_library(
+    name = "LlmInference",
+    module_name = "LlmInference",
+    generates_header = 1,
+    generated_header_name = "generated_header/LlmInference-Swift.h",
```

Comment:
@priankakariatyml Encountering error "The generated header of a Swift module may not contain directory components".

I'll drop the "generated_header/" prefix. Let me know if this leads to trouble on your side.

Reply:
That's okay. This was for my Objc Test. I am not checking them now. When I do, I'll check what the problem is